Assets/Scripts/RedRunner/Networking/RoundsManager.cs

```diff
@@ -48,6 +48,19 @@ public void RpcResetRound()
             GameManager.Singleton.StartGame();
             if (!Application.isBatchMode) {
                 GameManager.Singleton.RespawnMainCharacter();
+                GameManager.Singleton.LockCharacterToStart();
+                ServerSpawner.Instance.InitiateChoosing();
+            }
+        }
+
+        // receive on client to reset round
```

Comment:
receive on client to reset round -> "received on client to unblock character when choosing phase is over" or something along those lines